Updated Login Page to add new refresh token pointer (#3982)
MicrosoftDocs/azure-docs-cli: docs-ref-conceptual/authenticate-azure-cli-interactively.md

> az login -u $AzCred.UserName -p $AzCred.GetNetworkCredential().Password
> ```

## Refresh tokens

> When you sign in with a user account, Azure CLI generates and stores an authentication refresh token. When a access token is granted, it is only valid for a short amount of time. Therefore, a refresh token is also generated at the same time, so that the client application can exchange this refresh token for a new access token when needed. 

Reviewer: ```suggestion
When you sign in with a user account, Azure CLI generates and stores an authentication refresh token. When an access token is granted, it is only valid for a short amount of time. Therefore, a refresh token is generated at the same time, so that the client application can exchange an expired token for a new access token when needed. 
```

Use [az account get-access-token](/cli/azure/account#az-account-get-access-token) to retrieve the access token: 

```azurecli
az account show --output table

# get access token for the active subscription
az account get-access-token

# get access token for a specific subscription
az account get-access-token --subscription "<subscription ID or name>"

az account get-access-token --scope "myScope" --subscription "mySubscriptionName" --name "resourceGroupName"

```

> Starting from Azure CLI 2.54.0, `az account get-access-token` returns the `expires_on` property alongside the `expiresOn` property for the token expiration time. The `expires_on` property represents a Portable Operating System Interface (POSIX) timestamp while the`expiresOn` property represents a local datetime. We recommend for downstream applications to use the `expires_on` property, because it uses the Universal Time Code (UTC). However, it should be noted that the `expiresOn` property cannot express "fold" when Daylight Saving Time ends. This can cause problems in countries or regions where Daylight Saving Time is adopted. For more information on "fold", see [PEP 495 – Local Time Disambiguation](https://peps.python.org/pep-0495/).

Reviewer: ```suggestion
Starting from Azure CLI 2.54.0, `az account get-access-token` returns the `expires_on` property alongside the `expiresOn` property for the token expiration time. The `expires_on` property represents a Portable Operating System Interface (POSIX) timestamp while the`expiresOn` property represents a local datetime. We recommend for downstream applications to use the `expires_on` property, because it uses the Universal Time Code (UTC). 

The `expiresOn` property doesn't express "fold" when Daylight Saving Time ends. This can cause problems in countries or regions where Daylight Saving Time is adopted. For more information on "fold", see [PEP 495 – Local Time Disambiguation](https://peps.python.org/pep-0495/).
```

Reviewer: @daphnemamsft , This PR repeats the "starting from ..." message three times.  When repetition is needed, we use an /Include file.  I think we should do so in this instance as well.
